src/vs/workbench/api/common/extHostTreeViews.ts

```diff
@@ -977,6 +985,9 @@ export class ExtHostTreeView<T> extends Disposable {
 				parentNode.children = [];
 			}
 			parentNode.children.push(node);
+			// {{ SQL Carbon Edit }} Update Node in cache to make sure children are available when refreshing nodes later.
+			// This fixes issues during Refresh of Azure tree account nodes.
```

Comment:
What issues? And why would this only impact the Azure tree? I'd suggest trying to make this comment more descriptive of the problem in a general way - since from what I'm understanding this is something that would apply to all ext host trees.